ethpm_cli/parser.py

```diff
@@ -323,6 +336,14 @@ def registry_remove_cmd(args: argparse.Namespace) -> None:
 )
 registry_activate_parser.set_defaults(func=registry_activate_cmd)
 
+# ethpm registry explore
+registry_explore_parser = registry_subparsers.add_parser(
+    "explore", help="Explore a registry's list of released packages and manifest uris.",
+)
+add_uri_to_parser(
+    registry_explore_parser, "Registry URI for target registry.",
```

Comment:
Were we thinking of also supporting any aliased registries? I think it'd be more useful than requiring the entire uri everytime you want to explore a registry.  imo `ethpm registry explore ens` > `ethpm registry explore ethpm://ens.snakecharmers.eth/` for example. I think we do something similar for `ethpm registry activate`

Reply:
Ahh okay I see the desired behavior; I'll remove the error thrown directly in `explore_registry` and just use `resolve_uri_or_alias`